src/main/java/lotto/model/LottoNumbersGenerator.java

```diff
@@ -0,0 +1,38 @@
+package lotto.model;
+
+import static java.util.stream.Collectors.*;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Random;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoNumbersGenerator {
+	private static final int FROM_ZERO = 0;
+	private static final int ONE_FOR_EXCLUSIVE = 1;
+
+	private final Random random;
+
+	public LottoNumbersGenerator(Random random) {
+		this.random = random;
+	}
+
+	public List<LottoNumbers> generateRandomly(int count) {
+		return IntStream.range(FROM_ZERO, count)
+			.mapToObj((i) -> new LottoNumbers(generateRandomNumbers()))
+			.collect(Collectors.toList());
+	}
+
+	private List<Integer> generateRandomNumbers() {
+		List<Integer> candidateNumbers = generateLottoCandidateNumbers();
+		Collections.shuffle(candidateNumbers, random);
```

Comment:
random 객체를 외부에서 주입 받아서 써야 할 필요가 있을까요?

```suggestion
		Collections.shuffle(candidateNumbers);
```

Reply:
1. MockRandom을 주입 받아서 어떤 내용을 테스트 할 수 있는걸까요? 무작위로 값이 나오는걸 테스트하려는 의도일까요? 작성해주신 테스트 코드는 10개가 생성됐는지만 테스트 하고 있어서 의미 있는 테스트로 보기 어려울 것 같습니다.
2. 지금 단계에서는 스레드나 보안 걱정을 할 필요는 없어보입니다.

Reply:
class MockRandom extends Random {
		@Override
		public int nextInt(int bound) {
			return 1;
		}

		@Override
		protected int next(int bits) {
			return 2;
		}
	}
	
테스트에서 이런 클래스를 만들어서 LottoNumbersGenerator 에 주입하면 되지 않을까? 라는 생각을 했습니다.
그럼 무작위 숫자를 테스트 단계 에서 결정할 수 있으니까요.
근데 제가 실제로 저렇게 테스트도 하지 않고 말로만 MockRandom 이라고 말씀드려서 혼란스러우셨을거 같아요.
그리고 지금 Collections.shuffle 를 매소드로 무작위 숫자를 뽑기 때문에 저런식으로 테스트 하는 것도 억지스러운거 같습니다.

코멘트 주신대로 주입 받는 것을 빼고 static 으로 처리하도록 하겠습니다!